### PR TITLE
Fix license key padding

### LINKED ISSUE #3038: License error too close to input
Needs padding

Image

### PR DESCRIPTION
**Pull request by akshay-gupta7**
Resolves #3038 

Screenshot 2024-08-01 at 2 13 47 PM

### CODE REVIEW

File: packages/tokens-studio-for-figma/src/app/components/AddLicenseKey/AddLicenseKey.tsx [hunk 1]
```diff
@@ -129,16 +129,25 @@ export default function AddLicenseKey() {
...
+              />
+            )}
             name="license-key"
             data-testid="settings-license-key-input"
             value={newKey || ''}
             onChange={onLicenseKeyChange}
             validationStatus={licenseKeyError ? 'error' : undefined}
           />
-          {licenseKeyError && <ErrorMessage>{licenseKeyError}</ErrorMessage>}
+          {licenseKeyError && (
```
Comment (cuserox) [inline packages/tokens-studio-for-figma/src/app/components/AddLicenseKey/AddLicenseKey.tsx:146]: Can we "push things down" if possible? Therefore: `css={{ marginBottom: '$2' }}` on the input 🤝 

Or `css={{ paddingTop: '$2' }}` here.

Removes the potential of margins collapsing, which can happen when you have conflicting marginTop & marginBottom.

**Review by cuserox — CHANGES_REQUESTED** [review]
Small CSS tweak. Good to go after the change 🫰

File: packages/tokens-studio-for-figma/src/app/components/AddLicenseKey/AddLicenseKey.tsx (hunk 1 above)
Comment (akshay-gupta7) [reply packages/tokens-studio-for-figma/src/app/components/AddLicenseKey/AddLicenseKey.tsx:146]: Done!

**Review by cuserox — APPROVED** [review]
Thanks for making the change! 🥇

**Pull request merged**